spec/features/homepage_spec.rb

```diff
@@ -0,0 +1,6 @@
+feature 'Homepage' do
+  scenario 'has a title on the homepage' do
+    visit('/')
```

Comment:
you could make the chitter homepage route '/chitter' and make the messages route '/chitter/messages' as I think Eddie said it makes more sense for the user to visit those routes